testenv.Containerfile

```diff
@@ -148,8 +148,7 @@ RUN systemctl enable ion bpecho@4 ion-app-proxy && \
     mkdir -p /var/run/ion
 
 # Runtime config for this container
-COPY deps/dtnma-tools/integration-test-ion/node-*.rc /etc/ion/
-COPY deps/test-ion-configs/mgr.rc etc/ion/
+COPY deps/test-ion-configs/mgr.rc etc/ion/node-1.rc
```

Comment:
At one point there was some dynamic just-in-time config editing in the container entrypoint. Is that no longer needed?